conda-forge.yml

```diff
@@ -6,3 +6,4 @@ github:
   tooling_branch_name: main
 provider:
   linux_aarch64: default
+  linux_ppc64le: default
```

Comment:
JFYI this is the key line to enable `linux_ppc64le` builds. The rest is just from re-rendering